.github/workflows/sonarcloud.yml

```diff
@@ -65,7 +66,12 @@ jobs:
           sudo update-alternatives --set cc /usr/bin/gcc
           sudo update-alternatives --install /usr/bin/c++ c++ /usr/bin/g++ 30
           sudo update-alternatives --set c++ /usr/bin/g++
-
+      
+      - name: Compile Boost
+        uses: ./.github/workflows/compile-boost
+        with:
+          prefix: "${GITHUB_WORKSPACE}/deps"     
```

Comment:
Why ?

Reply:
boost version in our linux platforms are old, so i'm building the same version as windows.